<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10">
    <item>
     <layout class="QHBoxLayout" name="hTabs" stretch="0,0,0,1">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="twSim">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="SimulationFrame" name="tSim">
         <attribute name="title">
          <string>Simulation</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="twAberr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="AberrationFrame" name="tAberr">
         <attribute name="title">
          <string>Aberrations</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="twMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="TemFrame" name="tTem">
         <attribute name="title">
          <string>TEM</string>
         </attribute>
        </widget>
        <widget class="StemFrame" name="tStem">
         <attribute name="title">
          <string>STEM</string>
         </attribute>
        </widget>
        <widget class="CbedFrame" name="tCbed">
         <attribute name="title">
          <string>CBED</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="hs1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hPlots">
      <item>
       <widget class="QTabWidget" name="twReal">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="twRecip"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionGeneral"/>
    <addaction name="actionSet_area"/>
    <addaction name="actionAberrations"/>
    <addaction name="separator"/>
    <addaction name="actionExport_parameters"/>
    <addaction name="actionImport_parameters"/>
    <addaction name="separator"/>
    <addaction name="actionSimulate_EW"/>
    <addaction name="actionSimulate_Image"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOpenCL"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar::item{border: 0px}</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSimulate_EW">
   <property name="text">
    <string>Simulate EW</string>
   </property>
  </action>
  <action name="actionSimulate_Image">
   <property name="text">
    <string>Simulate Image</string>
   </property>
  </action>
  <action name="actionSet_area">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionOpenCL">
   <property name="text">
    <string>OpenCL</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionImportSettings">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExportSettings">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport_3">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport_parameters">
   <property name="text">
    <string>Export parameters</string>
   </property>
  </action>
  <action name="actionImport_parameters">
   <property name="text">
    <string>Import parameters</string>
   </property>
  </action>
  <action name="actionAberrations">
   <property name="text">
    <string>Aberrations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimulationFrame</class>
   <extends>QWidget</extends>
   <header>frames/simulationframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemFrame</class>
   <extends>QWidget</extends>
   <header>frames/temframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StemFrame</class>
   <extends>QWidget</extends>
   <header>frames/stemframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CbedFrame</class>
   <extends>QWidget</extends>
   <header>frames/cbedframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AberrationFrame</class>
   <extends>QWidget</extends>
   <header>frames/aberrationframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
